admin.go

```diff
@@ -380,3 +417,87 @@ func (ca *clusterAdmin) DeleteACL(filter AclFilter, validateOnly bool) ([]Matchi
 	}
 	return mAcls, nil
 }
+
+func (ca *clusterAdmin) DescribeConsumerGroup(group string) (*GroupDescription, error) {
+	controller, err := ca.client.Coordinator(group)
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := controller.DescribeGroups(&DescribeGroupsRequest{
+		Groups: []string{group},
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	if len(response.Groups) != 1 {
+		return nil, fmt.Errorf("Received unexpected number of results for request: %v", len(response.Groups))
+	}
+
+	return response.Groups[0], nil
+}
+
+func (ca *clusterAdmin) ListConsumerGroups() (allGroups map[string]string, err error) {
+	allGroups = make(map[string]string)
+
+	// Query brokers in parallel, since we have to query *all* brokers
+	brokers := ca.client.Brokers()
+	groupMaps := make(chan map[string]string, len(brokers))
+	errors := make(chan error, len(brokers))
+	wg := sync.WaitGroup{}
+
+	for _, b := range brokers {
+		wg.Add(1)
+		go func(b *Broker, conf *Config) {
+			defer wg.Done()
+			_ = b.Open(conf) // Ensure that broker is opened
```

Comment:
you can do 
```suggestion
			b.Open(conf) // Ensure that broker is opened
```

Reply:
i can, but then we don't document that there's an error (deliberately) ignored here.

Reply:
in fact errcheck will fail travis if i do what you suggest.. https://api.travis-ci.org/v3/job/490310704/log.txt i reverted that.